app/src/main/java/com/kickstarter/services/apiresponses/commentresponse/CommentEnvelope.kt

```diff
@@ -0,0 +1,35 @@
+package com.kickstarter.services.apiresponses.commentresponse
+
+import android.os.Parcelable
+import com.kickstarter.libs.qualifiers.AutoGson
+import com.kickstarter.models.Comment
+import kotlinx.android.parcel.Parcelize
```

Comment:
Nice touch using kotlin Parcelize  instead of the old java `auto.parcel.AutoParcel`
:heartbeat:
awesome work @sunday-okpoluaefe